apps/rush/src/utilities/PublishUtilities.ts

```diff
@@ -139,7 +150,7 @@ export default class PublishUtilities {
   }
 
   public static isRangeDependency(version: string): boolean {
-    const LOOSE_PKG_REGEX: RegExp = />=?(?:\d+\.){2}\d+\s+<(?:\d+\.){2}\d+/;
+    const LOOSE_PKG_REGEX: RegExp = />=?(?:\d+\.){2}\d+(\-[0-9A-Za-z-.]*)?\s+<(?:\d+\.){2}\d+/;
```

Comment:
> />=?(?:\d+\.){2}\d+(\-[0-9A-Za-z-.]*)?\s+<(?:\d+\.){2}\d+/; [](start = 35, length = 60)

:-o

Can you use [semver.validRange()](https://www.npmjs.com/package/semver) for this?

This is turning into a pretty complicated RegExp...  Not sure the 5 unit test cases are really checking all the edge cases.